src/Forum/client.ts

```diff
@@ -0,0 +1,49 @@
+import fetch, { Response } from 'node-fetch'
+import { env } from 'decentraland-commons'
+import { ForumPost } from './Forum.types'
+
+export type CreateSuccess = {
```

Comment:
Should we need to keep these types here or on `Forum.types.ts`?

Reply:
Good catch, probably on `.types.ts`, I meant to have them there and forgot